src/Functions/XPath/ASTs/ASTXPathText.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <Parsers/IAST.h>
+
+namespace DB
+{
+
+class ASTXPathText : public IAST
```

Comment:
Let's make it more general ASTXPathFunction and apply specific logic in the instruction.
This will simplify the support of the other xpath function later.